### PR TITLE
chore: update example apps after header based auth release

### PR DESCRIPTION
**Pull request by alisher-aituarov**
## Summary of change

Update example apps according to latest release 

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Documentation changes

(If relevant, please create a PR in our [docs repo](https://github.com/supertokens/docs), or create a checklist here highlighting the necessary changes)

## Checklist for important updates

-   [ ] Changelog has been updated
-   [ ] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
-   [ ] Changes to the version if needed
    -   In `package.json`
    -   In `package-lock.json`
    -   In `lib/ts/version.ts`
-   [ ] Had run `npm run build-pretty`
-   [ ] Had installed and ran the pre-commit hook
-   [ ] Issue this PR against the latest non released version branch.
    -   To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
    -   If no such branch exists, then create one from the latest released branch.
-   [ ] If added a new recipe interface, then make sure that the implementation of it uses NON arrow functions only (like `someFunc: function () {..}`).
-   [ ] If I added a new recipe, I also added the recipe entry point into the `size-limit` section of `package.json` with the size limit set to the current size rounded up.
-   [ ] If I added a new recipe, I also added the recipe entry point into the `rollup.config.mjs`

## Remaining TODOs for this PR

-   [ ] Item1
-   [ ] Item2

### CODE REVIEW

**Comment by porcellus** [comment]
Please add a comment about which examples you checked/ran locally.

**Comment by rishabhpoddar** [comment]
@alisher-aituarov whats the status on this issue? It's been pending since a while now.

**Comment by rishabhpoddar** [comment]
@alisher-aituarov @porcellus is this ready to merge?

**Review by porcellus — APPROVED** [review]

File: examples/with-next-iframe/config/frontendConfig.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ export const frontendConfig = () => {
             EmailPassword.init(),
             Session.init({
                 isInIframe: true,
+                tokenTransferMethod: "header",
```
Comment (rishabhpoddar) [inline examples/with-next-iframe/config/frontendConfig.js:16]: does this make the examlpe work on safari and incognito chrome? If yes, please update the readme in this example app cause it has a "### IMPORTANT:" section which needs to be removed.

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

**Review by rishabhpoddar — APPROVED** [review]

**Pull request merged**